homeassistant/components/simplefin/config_flow.py

```diff
@@ -0,0 +1,75 @@
+"""Config flow for SimpleFIN integration."""
+
+from typing import Any
+
+from simplefin4py import SimpleFin
+from simplefin4py.exceptions import (
+    SimpleFinAuthError,
+    SimpleFinClaimError,
+    SimpleFinInvalidAccountURLError,
+    SimpleFinInvalidClaimTokenError,
+    SimpleFinPaymentRequiredError,
+)
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigFlow, ConfigFlowResult
+
+from .const import CONF_ACCESS_URL, DOMAIN, LOGGER
+
+
+class SimpleFinConfigFlow(ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for the initial setup of a SimpleFIN integration."""
+
+    async def async_step_user(
+        self, user_input: dict[str, Any] | None = None
+    ) -> ConfigFlowResult:
+        """Prompt user for SimpleFIN API credentials."""
+        errors: dict[str, str] = {}
+
+        if user_input is not None:
+            access_url = user_input.get(CONF_ACCESS_URL, None)
+            if not access_url:
```

Comment:
The field is required, not sure why you made it optional. check how async_abort_entries match is used